kernel/update_kernel_fork.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -e
+script_name="${0##*/}"
+script_dir=$(dirname $(realpath -s "$0"))
+release_tool="${script_dir}/release-tool"
+
+die() {
+	echo >&2 -e "\e[1mERROR\e[0m: $*"
+	exit 1
+}
+
+info() {
+	echo -e "\e[1mINFO\e[0m: $*"
+}
+
+
+# Linux stable repository: From here will pull latest kernel
+# We can not pull from clear containers fork because is not automatically mirrored.
+linux_stable_repo=https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git 
+# Repository to push new clear containres kernel
+clear_containers_repo="https://github.com/clearcontainers/linux.git"
```

Comment:
Same comment - this is perfectly correct, but how about naming this `project_kernel_repo=` or similar?